modules/samples/product/FlickrClient/src/org/wso2/appserver/sample/flickr/client/FlickrServiceFrame.java

```diff
@@ -411,7 +411,7 @@
     private FlickrServiceClient client = new FlickrServiceClient();
     private String key;
     private String host = "api.flickr.com";
-    private String port = "80";
+    private String port = "443";
```

Comment:
value of the variable `port` is changed in [line 522](https://github.com/rasika90/product-as/blob/3fd52decc8b2c405a9b972bf9ce74b792312e685/modules/samples/product/FlickrClient/src/org/wso2/appserver/sample/flickr/client/FlickrServiceFrame.java#L522).